package.json

```diff
@@ -14,10 +14,17 @@
     "start": "yarn workspace mx-keycap-finder-server start",
     "dev:cypress": "yarn workspace mx-keycap-finder-server dev:e2e",
     "e2e:open": "yarn workspace mx-keycap-finder-e2e e2e:open",
-    "e2e:run": "yarn workspace mx-keycap-finder-e2e e2e:run"
+    "e2e:run": "yarn workspace mx-keycap-finder-e2e e2e:run",
+    "knex:migrate:latest": "yarn workspace mx-keycap-finder-server migrate:latest",
+    "knex:migrate:rollback": "yarn workspace mx-keycap-finder-server migrate:rollback",
+    "knex:migrate:make": "yarn workspace mx-keycap-finder-server migrate:make",
+    "db:seed": "yarn workspace mx-keycap-finder-server db:seed"
   },
   "engines": {
     "node": "^14.15",
     "yarn": "1.22.10"
+  },
+  "dependencies": {
+    "trim-newlines": "^4.0.2"
   }
```

Comment:
avoid adding dependencies to the root package.json. This is to make the disinction between libraries used in the backend and frontend, which are very distinct environments that we want to keep separate. If you want `trim-newlines` in Express and React, then add this dependency to both package.jsons for those subapps